WalletWasabi.Packager/Program.cs

```diff
@@ -495,10 +495,16 @@ private static void Publish()
 						$"/p:ErrorReport=none " +
 						$"/p:DocumentationFile=\"\" " +
 						$"/p:Deterministic=true",
-					WorkingDirectory = GuiProjectDirectory
+					WorkingDirectory = GuiProjectDirectory,
+					RedirectStandardOutput = true
 				}))
 				{
+					string error = process.StandardOutput.ReadToEnd();
```

Comment:
I believe, you should use `using` here as demonstrated in: https://github.com/zkSNACKs/WalletWasabi/pull/4086/files#diff-fe8eb3d00b7363de3f67b3be386d85b9R455

Reply:
Not that it's critical I guess.